client/keys/show.go

```diff
@@ -113,8 +110,6 @@ func runShowCmd(cmd *cobra.Command, args []string) (err error) {
 		printKeyAddress(info, bechKeyOut)
 	case isShowPubKey:
 		printPubKey(info, bechKeyOut)
-	case isShowMultiSig:
-		printMultiSigKeyInfo(info, bechKeyOut)
```

Comment:
Why are removing this btw?

Reply:
We don't need extra processing anymore to display a tree of multisig keys

Reply:
e.g. this comes out of the box

```
- name: multi
  type: multi
  address: cosmos195eza9hyjx9w4amt3he37hqdcv66tjx8uwpapw
  pubkey: cosmospub1ytql0csgqyfzd666axrjzq3zqh4r75vtgxx2cndqpgy60yc9tfu9xxgwmqqnshgkdw79wnuv8ufzd666axrjzq39tr0mg9xyvelgmq4qxvm9ytvf2flduzwfmhd62fkvpjunfmzg0u8h900d
  mnemonic: ""
  threshold: 1
  pubkeys:
  - address: cosmos1dj4ghqevu9y788gkc6gafdpgx6umh8qplffye4
    pubkey: cosmospub1addwnpepqg3qt63l2x95rr9vfksq5zd8jvz457znry8dsqfct5txh0zhf7xr7yssng9
    weight: 1
  - address: cosmos104ytdpvrx9284zd50v9ep8c6j7pua7dkk0x3ek
    pubkey: cosmospub1addwnpepqgj43ha5znzxvl5ds2srxdjj9ky4ylk7p8yamka9ymxqewf5a3y870kwj76
    weight: 1
```